qgreenland/config/layers/Glaciology/Ice sheet velocity/ESA Climate Change Initiative/layers.py

```diff
@@ -0,0 +1,67 @@
+from qgreenland.config.datasets.esa_cci import esa_cci_ice_sheet_velocity_20191214_20200131 as dataset
+from qgreenland.config.helpers.steps.build_overviews import build_overviews
+from qgreenland.config.helpers.steps.decompress import decompress_step
+from qgreenland.config.helpers.steps.warp_and_cut import warp_and_cut
+from qgreenland.config.project import project
+from qgreenland.models.config.layer import ConfigLayer, ConfigLayerInput
+
+
+_esa_cci_velocity_params = {
+    'esa_cci_velocity_magnitude': {
+        'title': 'Land ice surface velocity 2019-2020 (250m)',
+        'description': (
+            """Magnitude of horizontal ice velocity in meters per day. Data is
+            not masked to ice sheet boundary."""
+        ),
+        'variable': 'land_ice_surface_velocity_magnitude',
+        'style': 'esa_cci_velocity',
+    },
+    'esa_cci_velocity_vertical': {
+        # TODO: should this title aslo indicate 2019-2020?
+        'title': 'Land ice surface vertical velocity (250m)',
```

Comment:
I think we should make these titles consistent, e.g.:

`Land ice surface {horizontal|vertical} velocity 2019-2020 (250m)`